requirements.txt

```diff
@@ -1,4 +1,4 @@
-ansible>=2.7.8
+ansible==2.7.10
```

Comment:
Maybe:
`ansible==2.7.*`
or:
`ansible>=2.7.8,<2.8`

Reply:
`ansible>=2.7.8,<2.8` is best. Earlier versions have bugs.

Reply:
Updated submitted